src/components/QuantityInput/QuantityInput.js

```diff
@@ -28,8 +28,11 @@ function QuantityInput({ quantity, decreaseQuantity, increaseQuantity }) {
 }
 
 QuantityInput.propTypes = {
+  /** Specifies the quantity value */
   quantity: PropTypes.number.isRequired,
+  /** Event will trigger when decrease quantity button is clicked */
```

Comment:
`Event handler for onClick decrease quantity button callback.`

`decreaseQuantity` is not an Event. It is a handler.